### DISCUSSION TITLE
DBX use Pipfile instead requirements.txt

### DISCUSSION THREAD

**Discussion by joseph-freeman** [databrickslabs/dbx#179] [Q&A]
I am wondering if it is possible to use a Pipfile & Pipfile.lock when deploying a Databricks job. I noticed that there is a `--requirements-file` arg you can supply to the deploy, I didn't see anything equivalent for using a pipfile. Is this possible or is it not supported?

**Comment by renardeinside** [comment]
hi @joseph-freeman , 
please use `setup.py`-based dependency management approach. `requirements-file` support is deprecated in `dbx`.
